packages/Utils/test/CMakeLists.txt

```diff
@@ -1,5 +1,13 @@
 
 TRIBITS_ADD_EXECUTABLE_AND_TEST(
+  Miscellaneous_test
+  SOURCES tstMiscellaneous.cpp ${TEUCHOS_STD_UNIT_TEST_MAIN}
+  COMM serial mpi
```

Comment:
I am confused here.  The test is not enabled if I remove all together the `COMM` argument and will not run if I only specify `serial`.  With this current setting CTest calls `mpiexec -np 4 ...`